src/transformers/modeling_tf_bert.py

```diff
@@ -842,63 +926,104 @@ def call(self, inputs, **kwargs):
             >>> prediction_scores, seq_relationship_scores = outputs[:2]
 
         """
-        return_dict = kwargs.get("return_dict")
-        return_dict = return_dict if return_dict is not None else self.bert.return_dict
-        outputs = self.bert(inputs, **kwargs)
+
+        inputs = input_analysis(
+            func=self.call,
+            inputs=input_ids,
+            attention_mask=attention_mask,
+            token_type_ids=token_type_ids,
+            position_ids=position_ids,
+            head_mask=head_mask,
+            inputs_embeds=inputs_embeds,
+            output_attentions=output_attentions,
+            output_hidden_states=output_hidden_states,
+            return_dict=return_dict,
+            labels=labels,
+            next_sentence_label=next_sentence_label,
+            training=training,
+        )
+        return_dict = inputs["return_dict"] if tf.executing_eagerly() else self.bert.return_dict
```

Comment:
If `inputs["return_dict"]` is set to `False` in graph mode, an error or warning should be raised instead of silently ignoring it.